### ISSUE TITLE
maximum cooking time. what do you think if we add something like dropdown.

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/recipe-search-react#47]
I dont have an exact example in my hands right now. when will go home - will post it here

**Comment by chauhannishith** [comment]
I don't think that we should use keep any value here because it is not of much use as it is easier to type just two digits(I assume most people try to cook a dish which doesn't make them take more than 90 minutes:) ) instead of clicking and scrolling.

**Comment by atherdon** [comment]
i agree, let's keep it just numeric-only text input.
BUT!(as usually) - we need to solve this issue.
how we'll handle this value from a server side?


On 28-May-2018 at 01:26 PM, chauhannishith wrote:


I don't think that we should use keep any value here because it is not of
much use as it is easier to type just two digits(I assume most people try
to cook a dish which doesn't make them take more than 90 minutes:) )
instead of clicking and scrolling.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub
<https://github.com/ChickenKyiv/recipe-search-react/issues/47#issuecomment-392526690>,
or mute the thread

**Comment by atherdon** [comment]
so, when user fill max cooking time - we're using less or equal query,
right? because it can be a confusing thing sometime...


On 28-May-2018 at 01:26 PM, chauhannishith wrote:


I don't think that we should use keep any value here because it is not of
much use as it is easier to type just two digits(I assume most people try
to cook a dish which doesn't make them take more than 90 minutes:) )
instead of clicking and scrolling.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub
<https://github.com/ChickenKyiv/recipe-search-react/issues/47#issuecomment-392526690>,
or mute the thread

**Comment by atherdon** [comment]
did we have a validation/mask for numbers only at this field?
